client/components/common/AtAndBrowserDetailsModal/index.jsx

```diff
@@ -194,7 +194,26 @@ const AtAndBrowserDetailsModal = ({
                 );
         }
 
-        handleAction(updatedAtVersion, updatedBrowserVersion, updateMessage);
+        // Save full version even though we don't display it.
+        // Discard trailing zeros in found minor and patches.
+        // This also accounts for trailing numbers after the patch.
+        let fullVersion = '';
+        let [majorVersion, minorVersion, patch, ...rest] =
+            updatedBrowserVersion.split('.');
+
+        if (minorVersion === '0' && (patch === '0' || patch === undefined)) {
```

Comment:
```suggestion
        if (minorVersion === '0' && patch === '0') {
```

The `minorVersion` being `undefined` wasn't being handled which was causing cases of `{MAJOR}.undefined` being saved to the database.